app/main.py

```diff
@@ -2,0 +1,28 @@
+import os
+
+
+def move_file(command: str) -> None:
+    if len(command.split(" ")) != 3 or command.split(" ")[0] != "mv":
+        print("wrong command")
+        return
+
+    file_name, new_path = command.split(" ")[1:]
+
+    with open(file_name, "r") as sorce:
+        text = sorce.read()
+
+    os.remove(file_name)
```

Comment:
The code does not handle potential exceptions like `FileNotFoundError` or permission issues with `os.remove(file_name)`. Adding appropriate error handling is crucial.